tools/gen-version.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Creates the VERSION file in the LORIS root folder.
+set -euo pipefail
+
+# The name of the version file
+VERSION='VERSION'
+
+GIT_INSTALLED=$()
```

Comment:
it looks like you started to do things one way, and then changed your mind and did something different.. this doesn't seem to be used

Reply:
oops sorry, you're right.
```suggestion
```